backend/app/services/subscription/market_service.py

```
@@ -0,0 +1,651 @@
+            RentalSubscriptionResponse
+
+        Raises:
+            HTTPException: If validation fails
+        """
+        # Validate source subscription exists and is market visibility
+        source = (
+            db.query(Kind)
+            .filter(
+                Kind.id == source_subscription_id,
+                Kind.kind == "Subscription",
+                Kind.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)